pkgs/os-specific/darwin/rewrite-tbd/default.nix

```diff
@@ -13,4 +13,8 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [ cmake pkg-config ];
   buildInputs = [ libyaml ];
+
+  meta = with lib; {
+    platform = platforms.darwin;
```

Comment:
Can we add some more meta attributes while we change this?

Reply:
Added homepage attribute.

Reply:
Could you also add `meta.description`? It's not clear what this package does (even when looking at the homepage).